packages/frontend/pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>de.mayflower</groupId>
+    <artifactId>frontend</artifactId>
+    <version>0.0.1</version>
+
+    <name>frontend</name>
+    <description>Frontend for oTTo</description>
+
+    <properties>
+        <!-- plugin version -->
+        <plugins.openapi-generator.version>6.4.0</plugins.openapi-generator.version>
+    </properties>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <!-- source-generation from openapi -->
+                <plugin>
+                    <groupId>org.openapitools</groupId>
+                    <artifactId>openapi-generator-maven-plugin</artifactId>
+                    <version>${plugins.openapi-generator.version}</version>
+                    <executions>
+                        <execution>
+                            <goals>
+                                <goal>generate</goal>
+                            </goals>
+                            <phase>validate</phase>
+                            <configuration>
+                                <inputSpec>${project.basedir}/../api-spec/openapi.yml</inputSpec>
+                                <generatorName>typescript-axios</generatorName>
+                                <configOptions>
+                                    <npmName>rest-client</npmName>
+                                    <supportsES6>true</supportsES6>
+                                    <npmVersion>8.19.3</npmVersion>
+                                    <withInterfaces>true</withInterfaces>
+                                </configOptions>
+                                <output>${project.basedir}/generated-sources/client</output>
+                            </configuration>
+                        </execution>
+                    </executions>
+                </plugin>
+                <plugin>
+                    <groupId>com.github.eirslett</groupId>
+                    <artifactId>frontend-maven-plugin</artifactId>
+                    <!-- Use the latest released version:
+                    https://repo1.maven.org/maven2/com/github/eirslett/frontend-maven-plugin/ -->
```

Comment:
remove comment

Reply:
As the person who is most likely to work with that in the future, I find it helpful to keep the comments